packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -77,17 +79,44 @@ async function ensureLockedRecord(
   return attestationRecord
 }
 
+async function purgeExpiredRecords() {
+  const attestations = await kit.contracts.getAttestations()
+  const expiryTimeInSeconds = (await attestations.attestationExpiryBlocks()) * 5
+  const transaction = await sequelize!.transaction()
+  const AttestationTable = await getAttestationTable()
+  try {
+    await AttestationTable.destroy({
+      where: {
+        createdAt: {
+          [Op.lte]: moment()
+            .subtract(expiryTimeInSeconds, 'seconds')
+            .toDate(),
+        },
+      },
+      transaction,
+    })
+    await transaction.commit()
+  } catch (err) {
+    await transaction.rollback()
```

Comment:
Some logging here would be good. We'll want alerting for this

Reply:
ya -- i can add the sequelizeLogger in the transaction which seems to be how logging is handled in this service